controllers/translateController.test.js

```diff
@@ -496,8 +500,13 @@ describe("TranslateController", () => {
       [nameOfTranslation]: text,
     };
 
-    importJSONFile.mockImplementation(async (path) => {
-      if (path.includes("test-configs/test-config.json")) return testConfig;
+    importJSONFile.mockImplementation(async (filePath) => {
+      const normalizedPath =
+        platform === "win32"
+          ? pathModule.normalize(filePath).replace(/\\/g, "/")
+          : filePath;
```

Comment:
Maybe move this logic to a util called `normalizePath` with a JSDoc explaining what it does. So it can be reused each time it is needed (to avoid future problems with Windows) and we can scale it in case another OS has issues too.